submissions/blueprint-rant/blueprint-rant.md

```diff
@@ -0,0 +1,51 @@
+
+![](Generic tiling Solar Array blueprint)
+
+Balancers, *the black magic creations*, are another example of something that most players want to use, with fair reason, but don't want to learn and understand the maths and mechanics behind them. Figuring out the exact combination of belts, splitters and undergrounds to make the ratios perfect make sounds like a fantastic sunday afternoon for some people, but many would find it dull and unexciting. In both these scenarios, allowing the importing of blueprints will *likely* improve the players enjoyment of the game as they wont get stuck making monotonous blueprints or diving into the maths of balancers.
+
+![](Factorio balancer overview)
+
+### Blueprint library and copy/paste
+
+Once you have built a thing once in a world, its pretty common to need to build it again later, be it miner setups, train stops, train tracks, solar setups, bus layouts et cetera. This is especially compounded in multiplayer games, where each player using their own rail standards can cause chaos and destruction. In these situations, being able to copy, paste and share blueprints using the blueprint library, allows a much smoother and less chaotic experience. I believe the key difference here is, you are recreating soemthing you have already build in the given save, not importing something from somewhere else. As mentioned by *Rseding91* earlier, the ability to use blueprints can also so quickly ruin a multiplayer save, so the tradeoff here is important to think about.
+
+### Creativity in Factorio - Kovarex enrichment
+
+As a final note before my conclusion, I'd like to highlight just how much space for creativity there is in this game and the missed opportunities airising from blindly importing blueprints, using the [Kovarex enrichment process](https://wiki.factorio.com/Kovarex_enrichment_process). This late game uranium processing method allows you to increase the quantity of useful Uranium-235 using a cyclical process. I believe that it is probably one of the more blueprinted processes in the game due to the initial complexity it appears to have.
+
+![](Kovarex recipe from wiki)
+
+Recently on the [r/factorio](https://www.reddit.com/r/factorio/) subreddit, there was a flood of different Kovarex enrichment setup photos, each one trying to one up the last in creativity, efficiency and all out weirdness. I'd like to share a few of these here, to highlight just how many options there are in this game.
```

Comment:
I think you could add a few pictures here in addition to the links (don't know if that was your idea?). You could even combine all the designs into a single picture.

Reply:
I was planning on embedding the reddit posts or the images from the reddit posts. A collage may work better though so I'll have a look into that too. I'm open to other suggestions if someone can think of a better way of showing the different designs of though.

Reply:
I decided to settle for a collage + video for now unless someone can think of a better solution